### PR TITLE
Update brightness filter

### LINKED ISSUE #261: Update brightness filter
- Make brightness filter limited to numbers between 0-15

### PR DESCRIPTION
**Pull request by Xterionix**
Resolve #261

### CODE REVIEW

File: source/behavior/spawn_rules/components/brightness_filter.json [hunk 1]
```diff
@@ -8,12 +8,14 @@
     "min": {
       "type": "integer",
       "description": "This is the minimum light level value that allows the mob to spawn.",
-      "title": "Min"
+      "title": "Min",
+      "minimum": 0
```
Comment (DaanV2) [inline source/behavior/spawn_rules/components/brightness_filter.json:12]:
```suggestion
      "minimum": 0,
      "maximum": 0,
      "default": 0
```

File: source/behavior/spawn_rules/components/brightness_filter.json [hunk 2]
```diff
@@ -8,12 +8,14 @@
...
     },
     "max": {
       "type": "integer",
       "description": "This is the maximum light level value that allows the mob to spawn.",
-      "title": "Max"
+      "title": "Max",
+      "maximum": 15
```
Comment (DaanV2) [inline source/behavior/spawn_rules/components/brightness_filter.json:18]:
```suggestion
      "title": "Max",
      "maximum": 15
      "minimum": 0,
      "default": 15
```

**Review by DaanV2 — CHANGES_REQUESTED** [review]
Just ensuring you get the full range on all 😉

File: source/behavior/spawn_rules/components/brightness_filter.json (hunk 1 above)
Comment (Xterionix) [reply source/behavior/spawn_rules/components/brightness_filter.json:12]: I don't think max should be 0, probably just 15 too.

Comment (DaanV2) [reply source/behavior/spawn_rules/components/brightness_filter.json:12]: Yes my bad 😬

**Review by DaanV2 — APPROVED** [review]

**Pull request merged**